tools/ci_build/github/azure-pipelines/templates/android-qnn-java-api-aar.yml

```diff
@@ -0,0 +1,125 @@
+parameters:
+- name: buildConfig
+  displayName: Configuration to build
+  type: string
+  default: 'Release'
+
+- name: buildSettings
+  displayName: Path to the build setting file
+  type: string
+  default: ''
+
+- name: includedOpsConfig
+  displayName: Path to the included ops config file
+  type: string
+  default: ''
+
+- name: artifactName
+  displayName: Artifact Name
+  type: string
+  default: ''
+
+- name: job_name_suffix
+  displayName: Job name
+  type: string
+  default: ''
+
+- name: publish_executables
+  # The executables will only be published if the publish_executables is set to "1"
+  displayName: Publish executables such as onnxruntime_perf_test, ...
+  type: string
+  default: '0'
+
+- name: pool_name
+  displayName: Pool name
+  type: string
+  default: 'onnxruntime-Ubuntu2204-AMD-CPU'
+
+- name: packageName
+  displayName: Package Name
+  type: string
+  default: 'onnxruntime-android-qnn'
+
+- name: QnnSdk
+  displayName: QNN SDK version
+  type: string
+  default: 2.26.0.240828
```

Comment:
If we're not downloading the QNN SDK as part of this CI, letting the user specify a version doesn't seem necessary